src/overrides/venia-ui/components/ProductFullDetail/productFullDetail.js

```diff
@@ -215,45 +221,45 @@ const ProductFullDetail = props => {
                 <section className={classes.options}>{options}</section>
                 <section className={classes.quantity}>
                     <QuantityFields
-                        classes={{ root: classes.quantityRoot }}
+                        classes={{root: classes.quantityRoot}}
                         min={1}
                         message={errors.get('quantity')}
                     />
                 </section>
                 <section className={classes.actions}>
                     {cartActionContent}
                 </section>
-                {/*<section className={classes.description}>*/}
-                {/*    <span className={classes.descriptionTitle}>*/}
-                {/*        <FormattedMessage*/}
-                {/*            id={'productFullDetail.productDescription'}*/}
-                {/*            defaultMessage={'Product Description'}*/}
-                {/*        />*/}
-                {/*    </span>*/}
-                {/*    <RichContent html={product.short_description.html} />*/}
-                {/*</section>*/}
-
                 <section className={classes.tabsContainer}>
                     <TabGroup
```

Comment:
It doesn't seem really comfortably to pass tabs label and content through props, as currently the tabs functionality limited by descContent and attrContent

Would be better to think about implementation when you can specify tabs as much as you like

```
<TabGroup>
   <Tab label="Description">
          <section className={classes.description}>
              <span className={classes.descriptionTitle}>
                  <FormattedMessage
                      id={'productFullDetail.productDescription'}
                      defaultMessage={'Product Description'}
                  />
              </span>
              <RichContent html={product.short_description.html}/>
          </section>
   </Tab>
  <Tab label="Attributes">
    .....
  </Tab>
</TabGroup>
```

You can implement this differently, but just not like passing content through props. This is just an example